components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1656,25 +1618,19 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
                      call get_curr_date(year,month,day,tod)
                      !populate with corresponding pbuf
                      !variables
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_1_idx, aer_tau_bnd_lw_mon_1)
-                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_2_idx, aer_tau_bnd_lw_mon_2)
-                     if (current_month .ne. month) then
-                        aer_tau_bnd_lw_mon_1 = aer_tau_bnd_lw_mon_2
-                        if (month==12) then
+                     call pbuf_get_field(pbuf,aer_tau_bnd_lw_mon_idx, aer_tau_bnd_lw_mon)
+                     if (month==12) then
                            next_month = 1
                         else
                            next_month = month + 1
-                        end if
-                        allocate(aerosol_optical_property_lw(pcols,pver,nlwbands,begchunk:endchunk))
-                        call get_aerosol_optical_property_from_file(next_month,'AER_TAU_LW','lwband',nlwbands,aerosol_optical_property_lw)
-                        aer_tau_bnd_lw_mon_2 = aerosol_optical_property_lw(:,:,:,state%lchnk)
-                        deallocate(aerosol_optical_property_lw)
+                     end if
+                     if (current_month .ne. month) then
```

Comment:
It's my understanding that we don't really use `current_month` as a universal variable anymore.  It's only use is below for the interpolation, which could also simply use `month` right?  If so, I think we can get rid of `current_month` in it's current form everywhere.  If we like the term `current_month` then we can use that instead of `month` inside this subroutine.